docs/sections/user_guide/cli/tools/config.rst

```diff
@@ -284,229 +278,224 @@ and an additional supplemental YAML file ``values2.yaml`` with the following con
        date: 20240105
        empty: null
        greeting: Good Night
-       message: 'Good Night Moon Good Night Moon '
+       message: Good Night Moon Good Night Moon
        recipient: Moon
        repeat: 2
 
-* By default, variables/expressions that cannot be rendered are passed through unchanged in the output. For example, given config file ``config.yaml`` with contents
-
-  .. code-block:: yaml
-
-     roses: "{{ color1 }}"
-     violets: "{{ color2 }}"
-     color1: red
+* With the ``--dry-run`` flag specified, nothing is written to ``stdout`` (or to a file if ``--output-file`` is specified), but a report of what would have been written is logged to ``stderr``:
```

Comment:
```suggestion
* With the ``--dry-run`` flag specified, nothing is written to ``stdout`` (or content is saved to a file if ``--output-file`` is specified), but a report of what would have been written is logged to ``stderr``:
```

The original made it sound like nothing was written to a file if ``--output-file`` was specified. If that is the case maybe we can make this line clearer.

Reply:
I agree, will clarify.

Reply:
Oh, sorry, I misread the suggest and I actually disagree. I think the original PR text is accurate and grammatically correct.

Reply:
@NaureenBharwaniNOAA I mean, you're right about the meaning of the original text, but I believe this text just moved in the file and did not change, and I think the original text was correct: It says, IMO slightly less clumsily, that when `--dry-run` is specified, nothing is written to `stdout`; and, also, that if `--dry-run` is specified when `--output-file` is specified, nothing will be written to a file. Does that make sense?